iron-form.html

```diff
@@ -394,6 +422,9 @@
     },
 
     _doFakeSubmitForValidation: function() {
+      if (!!Polymer.clientDoesNotSupportFormValidationUI)
```

Comment:
But then i have an extra variable set on all the forms! Wasted Ks! (jk, will do, I don't care)